doc/sphinx/Pacemaker_Explained/fencing.rst

```diff
@@ -205,29 +226,22 @@ for ``pacemaker-fenced``.
    |                      |         |                    | be set.                                |
    +----------------------+---------+--------------------+----------------------------------------+
-   | pcmk_host_check      | string  | The default is     | .. index::                             |
-   |                      |         | ``static-list`` if |    single: pcmk_host_check             |
-   |                      |         | either             |                                        |
-   |                      |         | ``pcmk_host_list`` | How to determine which machines are    |
-   |                      |         | or                 | controlled by the device. Allowed      |
-   |                      |         | ``pcmk_host_map``  | values:                                |
-   |                      |         | is configured. If  |                                        |
-   |                      |         | neither of those   | * ``dynamic-list:`` query the device   |
-   |                      |         | are configured,    |   via the agent's ``list`` action      |
-   |                      |         | the default is     | * ``static-list:`` check the           |
-   |                      |         | ``dynamic-list``   |   ``pcmk_host_list`` or                |
-   |                      |         | if the fence       |   ``pcmk_host_map`` attribute          |
-   |                      |         | device supports    | * ``status:`` query the device via the |
-   |                      |         | the list action,   |   "status" command                     |
-   |                      |         | or ``status`` if   | * ``none:`` assume the device can      |
-   |                      |         | the fence device   |   fence any node                       |
-   |                      |         | supports the       |                                        |
-   |                      |         | status action but  |                                        |
-   |                      |         | not the list       |                                        |
-   |                      |         | action. If none of |                                        |
-   |                      |         | those conditions   |                                        |
-   |                      |         | apply, the default |                                        |
-   |                      |         | is ``none``.       |                                        |
+   | pcmk_host_check      | string  | Value appropriate  | .. index::                             |
+   |                      |         | to other           |    single: pcmk_host_check             |
+   |                      |         | parameters (see    |                                        |
+   |                      |         | "Default Check     | How to determine which machines are    |
```

Comment:
The description of ```pcmk_host_check``` was not as clear to me as ```pcmk_host_list``` or ```pcmk_host_map```, for example. Here's what I suggest as a possible improvement: "A value that determines which machines are controlled by the device. Allowed values: [...]".

Reply:
Good point. How about this: "The method Pacemaker should use to determine which nodes can be targeted by this device."